src/com_tjnotifications/admin/controllers/notification.php

```diff
@@ -26,24 +26,59 @@ class TjnotificationsControllerNotification extends \Joomla\CMS\MVC\Controller\F
 	/**
 	 * Function to add field data
 	 *
-	 * @return  void
+	 * @return  boolean
 	 */
 	public function editSave()
```

Comment:
3 methods in controller have same code.
Refactor those to have only one method.
You can generate code for same using this https://github.com/manojLondhe/joomla-extensions-generator